### PR TITLE
Added methods to get files based on the path when Drive mounted in Jupyter Lab

### PR DESCRIPTION
**Pull request by apdavison**
also implemented method to get sharing link

### CODE REVIEW

**Comment by Cracky5457** [comment]
hello Andrew, thanks for implementing that.

Sorry for the delay, I'm reviewing it right now and I will try to release it in the week.

File: ebrains_drive/file.py [hunk 1]
```diff
@@ -26,4 +28,31 @@ def get_file_by_url(self, file_url):
...
+        when the Drive is mounted in the EBRAINS Lab.
+        """
+        home_dir = "/mnt/user/drive/My Libraries/My Library/"
+        group_dir = "/mnt/user/drive/Shared with groups/"
+        shared_dir = "/mnt/user/shared/"
+        if local_path.startswith(home_dir):
+            repo_obj = self.client.repos.get_default_repo()
+            relative_path = os.path.relpath(local_path, home_dir)
+        elif local_path.startswith(group_dir):
+            collab_name = local_path.split("/")[5]
```
Comment (Cracky5457) [inline ebrains_drive/file.py:44]: I will replace it by a simple `local_path.replace(group_dir,"")` so we won't have issue if the amount of `/` is different if there is a `/` in the collab title.

**Review by Cracky5457 — CHANGES_REQUESTED** [review]

**Comment by Cracky5457** [comment]
Lab is actually not available to test it due to the infrastructure migration, I will apply the changes suggested in my review and push a new branch ( I can't commit on your PR it's in your own repo ) and release de .dev version to test it on the lab

**Comment by appukuttan-shailesh** [comment]
@Cracky5457 : would it be possible to review this and make a new release (0.2.1 ?) of `ebrains-drive` before this weekend?
We have the CodeJam event next week, and we are planning a small demo of this library. It would be nice to work with the latest release, and to be able to test it prior to the demo.

**Comment by Cracky5457** [comment]
Merged and fixed on master directly

**Pull request closed without merging**